.gitignore

```diff
@@ -6,11 +9,15 @@
 *.vs
 *CMakeLists.txt.user
 *vcxproj.user
-Compressonator/Applications/_Plugins/C3DModel_viewers/glTF_DX12_EX/GeneratedFiles/*
-Compressonator/Applications/_Plugins/C3DModel_viewers/glTF_OpenGL/GeneratedFiles/*
-Compressonator/Applications/CompressonatorCLI/Make/buildCLI_linux_package.sh
-Compressonator/Applications/CompressonatorGUI/GeneratedFiles/Debug_MD/*
-Compressonator/Applications/CompressonatorGUI/GeneratedFiles/qrc_CompressonatorGUI.cpp
-Compressonator/Applications/CompressonatorGUI/GeneratedFiles/Release_MD/*
-Compressonator/Build/*
+Applications/_Plugins/C3DModel_viewers/glTF_DX12_EX/GeneratedFiles/*
+Applications/_Plugins/C3DModel_viewers/glTF_OpenGL/GeneratedFiles/*
+Applications/CompressonatorCLI/Make/buildCLI_linux_package.sh
+Applications/CompressonatorGUI/GeneratedFiles/Debug_MD/*
+Applications/CompressonatorGUI/GeneratedFiles/qrc_CompressonatorGUI.cpp
+Applications/CompressonatorGUI/GeneratedFiles/Release_MD/*
+build-out/*
+build/*
```

Comment:
It's more important now to care about case sensitive paths. Additionally, I feel anything in the `Build` directory should actually go in `scripts`

Reply:
@UebelAndre  Agreed, build directory has been removed in current dev source (currently not posted). Users can now select where to build the applications. Build management is in scripts folder which is accompanied by cmakelists in the root directory.